pkgs/development/python-modules/piano-transcription-inference/default.nix

```
@@ -0,0 +1,74 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchPypi
+, fetchpatch
+, fetchurl
+, librosa
+, matplotlib
+, mido
+, torch
+, torchlibrosa
+}:
+
+buildPythonPackage rec {
+  pname = "piano-transcription-inference";
+  version = "0.0.5";
```

```suggestion
  version = "0.0.5";
  format = "setuptools";
```

Went missing apparently.